README.Rmd

```diff
@@ -31,3 +31,146 @@ You can install the development version from [GitHub](https://github.com/) with:
 # install.packages("devtools")
 devtools::install_github("szymanskir/hackeRnews")
 ```
+
+## Examples
+
+### Word cloud
```

Comment:
I suggest changing this title, we want to come from the problems perspective. We are not showing how to create a wordcloud :) Maybe something like "Check what buzzwords are used in Job Offers of Hacker News?"